test/Web/ScottySpec.hs

```diff
@@ -1,33 +1,36 @@
-{-# LANGUAGE OverloadedStrings, CPP #-}
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE OverloadedStrings #-}
+
 module Web.ScottySpec (main, spec) where
```

Comment:
minor functional change by moving some helpers from top-level into a `let` bind. This looks like a huge diff, but set github to ignore whitespace, as the changes are mostly indentation.